src/components/radio-group/radio-group.bemhtml

```diff
@@ -0,0 +1,27 @@
+block('radio-group')(
```

Comment:
because we have parents https://github.com/awinogradov/bem-components/blob/v2/common.blocks/radio-group/radio-group.bemhtml#L10-L37. Now our template is deoptimized but we should use applyNext and replace content from previous executions